### PR TITLE
Swagger docs including static swaggerui files

### PR DESCRIPTION
**Pull request by merlante**
This provides swagger documentation for the [v1.1 (alpha) version](https://github.com/RedHatInsights/authz/blob/main/api/v1alpha/openapi-authz-v1_alpha.yaml) of the openapi spec. Everything is static for now. This PR may be completely superseded by a PR (coming soon) that generates all the things from .proto (although I guess the assets will still be needed).

We can merge this if we're in need of something quickly.

### CODE REVIEW

**Comment by merlante** [comment]
If we do merge this. The steps to update the docs (for now) are:
* Edit openapi-authz-v1_alpha.yaml.
* Convert manually to json.
* Replace api/swaggerui/swagger.json with newly converted json.

File: host/Web.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ type Web struct {
 
 // Host blocks processing web requests until it ends, at which point it signals the indicated WaitGroup
 func (web Web) Host(wait *sync.WaitGroup) {
+	fs := http.FileServer(http.Dir("./api/swagger-ui"))
+	http.Handle("/swaggerui/", http.StripPrefix("/swaggerui/", fs))
```
Comment (DGuhr) [inline host/Web.go:24]: wdyt about using `/v1alpha/swaggerui` instead? and perhaps to change Web.go line 26 to also be `/v1alphapermissions/check` instead, so we're consistent?

**Review by DGuhr — CHANGES_REQUESTED** [review]
I tried it locally and this is shown calling /swaggerui currently:
Screenshot 2023-02-28 at 13 36 11

Could you remove the "User" part which is a duplication it seems?

Also, see code comment.

File: api/swagger-ui/swagger.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+             "url": "http://<TODO>"
+          }
+       },
+       {
+          "name": "user",
+          "description": "Operations about user"
+       }
+    ],
+    "paths": {
+       "/v1/permissions/check": {
```
[the remark this answers is not in the document]
Comment (merlante) [reply api/swagger-ui/swagger.json:40]: Same comment as above.

**Comment by merlante** [comment]
In general, there's also no point in fixing things that would require a change to the yaml and then manual regeneration, unless we have have a reason. If <customer> wants something ASAP and it's not good enough, then yes. Otherwise, generated files should be with us very soon.

**Comment by DGuhr** [comment]
- as long as its fixed in the real one, np for me. 
- as you said, no idea if we need this PR, waiting for feedback from others.

File: host/Web.go (hunk 1 above)
Comment (wscalf) [reply host/Web.go:24]: Personal opinion: adding something to the route that says it's alpha makes a ton of sense ... after code generation is in. The main benefit is being able to tinker freely like that. :)

**Comment by akoserwal** [comment]
@merlante instead of committing the swagger file, we can use docker image: https://github.com/RedHatInsights/authz/pull/32

**Pull request closed without merging**